src/js/modules/scrollToAnchor.js

```diff
@@ -0,0 +1,22 @@
+import { closeMenu } from './mobileMenu.js';
+
+const anchors = Array.from(document.querySelectorAll("a[href*='#']"));
```

Comment:
all your menu links have class `nav__link`, use it in querySelectorAll

Reply:
There are nav__link-footer in footer section and they have other styles. So I need already used selector.

Reply:
```js
headerMenu.querySelectorAll(".nav__link");
```

Reply:
I don't understand you. Why only headerMenu? I have menu links not only in header but also in footer. 
![image](https://github.com/IlyaDevN/Horse-club/assets/106959524/443ae5f2-c411-409e-a849-8d1241d9f0b5)
![image](https://github.com/IlyaDevN/Horse-club/assets/106959524/70ca2aa4-f7b8-4f41-863c-73f1d5f875f1)

Reply:
I didn't pay attention that footer link lead to this section as well. Then 
```js
document.querySelectorAll(".nav__link-footer");
```
and join all the links in one collection

I don't like the idea to find all the anchors with such a selector `a[href*='#']`
To control more what you do and avoid all the possible bugs, just find exactly the elements you need.

Reply:
Done